packages/kotti-ui/source/next/kotti-field-toggle/components/ToggleInner.vue

```diff
@@ -66,5 +66,9 @@ export default defineComponent({
 		color: var(--text-05);
 		cursor: not-allowed;
 	}
+
+	> *:not(:first-child) {
```

Comment:
what's that about

Reply:
I moved it from the `ToggleBox.vue` and `ToggleSwitch.vue` components and made it 25% smaller

Thought it’s better that way so that we can also style the tooltip with the same rule and avoid the duplication of the two files both having the same margin-right